rules/rules-reviewed/jakarta-ee9/java-ee/spring-components.windup.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0"?>
+<ruleset id="spring-components" xmlns="http://windup.jboss.org/schema/jboss-ruleset"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This ruleset ensures the versions of Spring and Spring Boot are Jakarta EE 9 compliant
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,3.0.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-java,3.0.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-xml,3.0.0.Final" />
+        </dependencies>
+        <targetTechnology id="jakarta-ee" versionRange="[9,)" />
+        <targetTechnology id="jws" versionRange="[6,)" />
```

Comment:
I feel like we should consider having a `jws6` target alias (just like we have the `hibernate6` one) to define which are the relevant targets for such a migration (e.g. `jakarta-ee`).
In this way we could avoid "mixing" JWS migration with Spring/Spring Boot/Jakarta migrations